### PR TITLE
Adding some tutorial on publishing

### PR DESCRIPTION
**Pull request by jeremyrickard**
Adding a little bit to describe the publish process, with some interactivity

This also incidentally updates the examples to have bundle tags.

### CODE REVIEW

File: docs/content/slides/pack-your-bags/index.md [hunk 1]
```diff
@@ -1174,6 +1191,72 @@ Custom mixins are just easier to use, but aren't necessary.
...
+* Run `porter build`
+* Run `porter publish`
+
+Example (assuming your username is cnabaholic):
+
+* Change `deislabs/cool-image:latest` to `cnabaholic/cool-image:latest`
+
+---
+
+# Try to run your neighbor's bundle
```
Comment (carolynvs) [inline docs/content/slides/pack-your-bags/index.md]: How safe is it to ask people to run arbitrary docker containers from people they don't know? I'd be leary of it though I do see what it is trying to show in the workshop.

File: docs/content/slides/pack-your-bags/index.md [hunk 2]
```diff
@@ -408,6 +409,19 @@ class: center, middle
 
 ---
 
+# How does it do that?
```
[the remark this answers is not in the document]
Comment (jeremyrickard) [reply docs/content/slides/pack-your-bags/index.md]: OK.

File: docs/content/slides/pack-your-bags/index.md (hunk 1 above)
Comment (jeremyrickard) [reply docs/content/slides/pack-your-bags/index.md]: Probably a good point. I guess we could have them just change to another directory and run it from the tag they just pushed or remove this step entirely.

File: docs/content/slides/pack-your-bags/index.md [hunk 3]
```diff
@@ -1174,6 +1191,72 @@ Custom mixins are just easier to use, but aren't necessary.
...
+tag: deislabs/porter-azure-wordpress-bundle:latest
+```
+
+---
+
+## porter publish
+
+```console
+$ porter publish --help
+Publishes a bundle by pushing the invocation image and bundle to a registry.
```
[the remark this answers is not in the document]
Comment (jeremyrickard) [reply docs/content/slides/pack-your-bags/index.md]: OK

**Pull request merged**